src/evo/dmnstate.h

```diff
@@ -324,7 +330,7 @@ class CDeterministicMNStateDiff
     DMN_STATE_DIFF_LINE(pubKeyOperator)                \
     DMN_STATE_DIFF_LINE(keyIDVoting)                   \
     DMN_STATE_DIFF_LINE(addr)                          \
-    DMN_STATE_DIFF_LINE(scriptPayout)                  \
+    DMN_STATE_DIFF_LINE(payoutShares)                  \
```

Comment:
Oh, nice catch! It is not that simple indeed since `CDeterministicMNStateDiff`  does not use the serialization of `CDeterministicMNState`, but I guess we can use the same trick we did with `pubKeyOperator`

Reply:
ok, now every `std::vector<PayoutShare>` is serialized with the same helper class (whose compatibility with older versions is tested in trivially_valid and trivially_invalid unit tests).

`CDeterministicMNStateDiff` also uses this class + from now on each time it serializes a vector of PayoutShare it serializes the version.